cudaungappedextender/samples/device_sample.cu

```diff
@@ -0,0 +1,142 @@
+/*
+* Copyright 2020 NVIDIA CORPORATION.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#include <cuda_runtime_api.h>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include <claraparabricks/genomeworks/cudaungappedextender/cudaungappedextender.hpp>
+#include <claraparabricks/genomeworks/io/fasta_parser.hpp>
+#include <claraparabricks/genomeworks/utils/cudautils.hpp>
+
+using namespace claraparabricks::genomeworks;
+using namespace claraparabricks::genomeworks::cudaungappedextender;
+
+int main(int argc, char* argv[])
+{
+    const int32_t input_xdrop      = 10;
+    const int32_t input_no_entropy = 0;
+    const int32_t hsp_threshold    = 20000;
```

Comment:
The values are 
`const int32_t input_xdrop      = 910;`
`const bool input_no_entropy = false;`
`const int32_t hsp_threshold    = 3000;`

Reply:
Changed.